Lib/fontbakery/profiles/fvar.py

```diff
@@ -607,3 +607,59 @@ def com_adobe_fonts_check_varfont_distinct_instance_records(ttFont, has_name_tab
 
     if passed:
         yield PASS, "All instance records are distinct."
+
+
+@check(
+    id = 'com.adobe.fonts/check/varfont/foundry_defined_tag_name',
+    rationale = """
+        According to the Open-Type spec's
+        syntactic requirements for foundry-defined design-variation
+        axis tags available at
+        https://learn.microsoft.com/en-us/typography/opentype/spec/dvaraxisreg
+
+        Foundry-defined tags must begin with an uppercase letter (0x41 to 0x5A),
+        and must use only uppercase letters or digits.
+    """,
+    conditions = ['is_variable_font'],
+    proposal = 'https://github.com/googlefonts/fontbakery/issues/4043'
+)
+def com_adobe_fonts_check_varfont_foundry_defined_tag_name(ttFont):
+    """Validate that foundry-defined tags begin with an uppercase letter (0x41 to 0x5A),
+       and use only uppercase letters or digits.
+       Warn if foundry-defined tag is similar to registered tag, aka is
+       equivalent to registered tag when using all lowercase letters.
+    """
+    registeredTags = ["ital", "opsz", "slnt", "wdth", "wght"]
```

Comment:
It might be worth to move this to [constants.py](https://github.com/googlefonts/fontbakery/blob/main/Lib/fontbakery/constants.py)

Reply:
It might also be ok to keep it here for now, until a second check actually needs to use it.

Reply:
I moved it to `constants.py` so that it'll be easier to find it & add checks related to this in the future! 🙂